octobot/initializer.py

```diff
@@ -27,9 +29,12 @@ def __init__(self, octobot):
 
     async def create(self):
         # initialize tentacle configuration
-        tentacles_config_path = self.octobot.get_startup_config(constants.CONFIG_KEY, dict_only=False).\
-            get_tentacles_config_path()
+        config = self.octobot.get_startup_config(constants.CONFIG_KEY, dict_only=False)
+        tentacles_config_path = config.get_tentacles_config_path()
         self.octobot.tentacles_setup_config = tentacles_manager_api.get_tentacles_setup_config(tentacles_config_path)
 
+        # check if tentacles setup requires an update
+        await commands.update_tentacles_if_necessary(self.octobot.tentacles_setup_config, config)
```

Comment:
Why here and not in cli ?

Reply:
There is no `tentacles_setup_config` in cli